code/modules/bitrunning/virtual_domain/domains/beach_bar.dm

```diff
@@ -13,10 +13,18 @@
 	desc = "Whose drink is this? Not yours, that's for sure. Well, it's not like they're going to miss it."
 	list_reagents = list(/datum/reagent/consumable/ethanol/pina_colada = 30)
```

Comment:
You didn't remove `/obj/item/reagent_containers/cup/glass/drinkingglass/filled/virtual_domain`

Reply:
i want to blame merge artifacts here but I don't even think I can tbh